frontend/src/features/admin-form/settings/components/MultiLanguageSection/FormMultiLanguageToggle.tsx

```diff
@@ -74,6 +78,15 @@ const LanguageTranslationRow = ({
     [mutateFormSupportedLanguages, supportedLanguages],
   )
 
+  const handleLanguageTranslationEditClick = useCallback(
+    (language: Language) => {
+      navigate(
+        `${ADMINFORM_ROUTE}/${formId}/settings/multi-language/${language.toString()}`,
```

Comment:
1. Possible to move this to `frontend/src/features/public-form/utils/urls.ts` since this might be a little complex and we might want to have some type-safety on the `language` variable. 

2. Looking at the `multi-language` path again, I feel that we could use `/language` instead of `/multi-language` since we're manipulating the "language" settings.